### PR TITLE
feat: camel modifier for `v-bind`

### LINKED ISSUE #37: camel modifier for `v-bind`
## About
doc: https://vuejs.org/api/built-in-directives.html#v-bind

### PR DESCRIPTION
**Pull request by LittleSound**
close #37

```vue
<svg :view-box.camel="viewBox"></svg>
<!-- compiled as: -->
<svg viewBox="xxx"></svg>
```

### CODE REVIEW

File: packages/compiler-vapor/src/transforms/vBind.ts [hunk 1]
```diff
@@ -21,6 +22,19 @@ export const transformVBind: DirectiveTransform = (dir, node, context) => {
     exp.ast = null
   }
 
+  if (modifiers.includes('camel')) {
+    if (arg.type === NodeTypes.SIMPLE_EXPRESSION) {
```
Comment (sxzz) [inline packages/compiler-vapor/src/transforms/vBind.ts:26]: `arg.type` is always a simple expression. Compound expression is not existing in Vapor compiler.

File: packages/compiler-vapor/src/generate.ts [hunk 2]
```diff
@@ -370,9 +372,11 @@ function genOperation(oper: OperationNode, context: CodegenContext) {
 }
 
 function genSetProp(oper: SetPropIRNode, context: CodegenContext) {
-  const { push, pushWithNewline, vaporHelper } = context
+  const { push, pushWithNewline, vaporHelper, helper } = context
   pushWithNewline(`${vaporHelper('setAttr')}(n${oper.element}, `)
+  if (oper.camel) push(`${helper(helperNameMap[CAMELIZE])}(`)
```
Comment (sxzz) [inline packages/compiler-vapor/src/generate.ts:377]: It should be a string literal.
```suggestion
  if (oper.camel) push(`${helper('camelize')}(`)
```

File: packages/compiler-vapor/__tests__/compile.test.ts [hunk 3]
```diff
@@ -130,11 +132,40 @@ describe('compile', () => {
       })
 
       // TODO: camel modifier for v-bind
```
Comment (sxzz) [inline packages/compiler-vapor/__tests__/compile.test.ts:134]: Remove todo

**Review by sxzz — COMMENTED** [review]
Left few comments for you.

**Comment by sxzz** [comment]
I refactored unit tests via 5f769745fa927274db86b68d0406f6f8b0d84e47. Could you please complete the unit tests I added to that commit?

**Comment by LittleSound** [comment]
Sure, I've finished those tests.

**Review by sxzz — APPROVED** [review]

**Pull request merged**